packages/flutter/lib/src/material/slider.dart

```diff
@@ -51,7 +51,10 @@ import 'theme.dart';
 /// given unbounded constraints, it will attempt to make the rail 144 pixels
 /// wide (with margins on each side) and will shrink-wrap vertically.
 ///
-/// Requires one of its ancestors to be a [Material] widget.
+/// Both [Material] and [MediaQuery] widgets are required to be present as
+/// ancestors for this widget. Typically, these are introduced by the
+/// [MaterialApp] or [WidgetsApp] widget at the top of your application widget
+/// tree.
```

Comment:
[Material] isn't introduced by either of those.

Reply:
Oh, I thought they were (especially the MaterialApp). Reworded.